barretenberg/cpp/src/barretenberg/vm2/tracegen/sha256_trace.cpp

```diff
@@ -299,6 +295,7 @@ void Sha256TraceBuilder::process(
                       } });
 
             // Computing W
+            // Can we replace this inside the if
```

Comment:
??

Reply:
whoops, expanded on what i meant to say here